lib/validators/movie.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).required(),
```

Comment:
nit: I don't think `.min(1)` is necessary here since `Joi.string()` doesn't allow the empty string. Or that may be the case on API.